proto/autokitteh/builds/v1/svc.proto

```diff
@@ -29,7 +29,7 @@ message SaveRequest {
     expression: "has(this.build) && this.build.build_id == ''"
   };
 
-  builds.v1.Build build = 1;
```

Comment:
how did it work before?

Reply:
which way is preffered? `Build` vs. `builds.v1.Build` ? In our proto code we are using the both, so I'm trying to add more consistency to the files I'm touching